lessons/Data-Science/variability/langs/en-us/pages/animal-weight-variability.adoc

```diff
@@ -0,0 +1,71 @@
+= Variability of Animals' Weights
+
+@link-instructions{
+The folks at the shelter want to approximate the amount of food they need to purchase for the coming month. They know there is a relationship between an animal's weight and how much it eats, so they are discussing the distribution of animals' weights. Help them out by responding to the prompts below. For question 7, you'll need to use the @starter-file{expanded-animals-sub-pops}.
+}
+
+@n Imagine a _typical_ animal from each of these four species, and estimate its weight. (To get you started, we've provided a weight estimate for a typical dog.) @hspace{2em} dog: @fitb{7em}{55 lbs} @hspace{2em} rabbit: @fitb{7em}{}  @hspace{2em} cat: @fitb{7em}{} @hspace{2em} tarantula: @fitb{7em}{}
+
+@n Circle the species with the _greatest_ variability in weight: @hspace{2em} @ifsoln-choice{dog @hspace{5em}} rabbit @hspace{5em} cat @hspace{5em} tarantula
+
+@n Circle the species with the _least_ variability in weight: @hspace{3em} dog @hspace{5em} rabbit @hspace{5em} cat @hspace{5em} @ifsoln-choice{tarantula}
+
+@n Below are dot plots for rabbits, cats, and tarantulas--but they aren't labelled! Write the species that you think is displayed on each plot.
+
+[cols="<.^1a, <.^1a, 1a"]
+|===
+
+| species: @fitb{}{@ifsoln{rabbit}}
+
+| species: @fitb{}{@ifsoln{tarantula}}
+
+| species: @fitb{}{@ifsoln{cat}}
+
+| @image{../images/rabbit-dot-plot.png, 350}
+
+| @image{../images/tarantula-dot-plot.png, 350}
+
+| @image{../images/cat-dot-plot.png, 300}
+
+|===
+
+@n Explain why your species labels (above) make sense. @fitb{}{}
+
+@fitb{}{@ifsoln{Of the 3 species, cats weigh the most and their weights have the most variability, while tarantulas weigh the least and also have the least variability.}}
+
+@fitb{}{}
+
+@n Imagine what the dot plot showing the distribution of dogs' weights will look like. Describe the dot plot, discussing peaks, gaps, outliers, and variability. It is okay if your prediction is off! @fitb{}{}
```

Comment:
@retabak Feels like students would be more successful here if we made `fitb{}{}` for each thing we want them to predict. Any reason not to ask for the same information here as in the table below? 

- the table below you also asks for range and typical weight
- the table below doesn't ask about peaks or variability

Reply:
decided to simplify things; removed the question about dogs weights because it was redundant; updated the table to say "range / variability".